app/src/org/commcare/views/media/AudioController.java

```diff
@@ -28,24 +31,24 @@
      * Button that corresponds to the currentEntity media. Pressing the button
      * should trigger playback control methods here.
      */
-    private AudioButton currentButton;
+    private AudioPlaybackReset currentAudioReset;
```

Comment:
It was needed during an intermediate state of class refactoring.
Generally I'm in favor of using interfaces to pair-down a class to the collection of minimally needed methods. In this case, `AudioPlaybackButton` only had 1 other public method, so that benefits of using an interface didn't over-weigh the mental overhead of the indirection, so I'll take it out.